bootstrap

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 set -euo pipefail
 
-APP_PARTS=(${_HANDLER//\// })
-tmpfile=`mktemp`
-echo "(require '${APP_PARTS[0]})" | cat - /opt/runtime.cljs > $tmpfile
+bin_dir=${BIN_DIR:-/opt}
+app_parts=(${_HANDLER//\// })
 
-/opt/lumo $tmpfile
\ No newline at end of file
+"$bin_dir/lumo" -e "(require (quote ${app_parts[0]}))" -m runtime
```

Comment:
Hi @arichiardi 
I just tried your fork of the project (including using your musl-based build of `lumo`), and got it working after tweaking the `bootstrap` file a bit:
```
"$bin_dir/lumo" --classpath ".:${bin_dir}" -e "(require (quote ${app_parts[0]}))" -m runtime
```
If I don't include the $bin_dir explicitly, it cannot seem to find the `runtime.cljs`. Additionally, I have to (re-)add current dir to the classpath.

Can you test that change? I can make a PR to your fork if you want me to.

Reply:
@arichiardi Btw, I got S3 access working with your Lumo build. Great work! :-) I'll eventually add an example to this repo.

Reply:
I would like myself to PR against your work if you want me too I guess, I don't mind either choice, let's pick one fork 😁😁

I will definitely try to add your change, I thought I had it working properly there but maybe not 🙃

Reply:
Oh I see will fix things

Reply:
Ok I changed this and rebase @grav. Sorry didn't have time for doing an end-to-end test but I think that if you did it we are probably good to proceed and fix things afterwards.

In any case your change makes definitely sense to me, so much that I don't know why it was not like that already before.